bindata/network/network-metrics/002-daemonset.yaml

```diff
@@ -0,0 +1,74 @@
+---
+kind: DaemonSet
+apiVersion: apps/v1
+metadata:
+  name: network-metrics-daemon
```

Comment:
The name "network-metrics-daemon" makes this sound much more generic than it really is. If this is really only about providing additional metrics for pods using multus, it should have a name that sounds more like that. (Likewise the https://github.com/openshift/network-metrics-daemon project)

Reply:
As per my comment above, the general agreement was to use this to collect other metrics too, hence the name.

Reply:
Once other metrics come into play this is probably also a conversation we should have with upstream sig-instrumentation.